cogboard-app/src/main/kotlin/com/cognifide/cogboard/widget/WidgetIndex.kt

```diff
@@ -13,6 +13,7 @@ class WidgetIndex {
             JenkinsJobWidget::class.java.simpleName -> JenkinsJobWidget(vertx, config)
             SonarQubeWidget::class.java.simpleName -> SonarQubeWidget(vertx, config)
             ServiceCheckWidget::class.java.simpleName -> ServiceCheckWidget(vertx, config)
+            TextWidget::class.java.simpleName -> TextWidget(vertx, config)
```

Comment:
In my opinion no backend is required for this widget.

Reply:
`updateState` method used to update properties from edit window.